src/btree/bt_debug.c

```diff
@@ -799,36 +780,36 @@ __wt_debug_disk(
     flags = dump_app_data ? WT_DEBUG_UNREDACT : 0;
     WT_RET(__debug_config(session, ds, ofile, flags));
 
-    WT_ERR(ds->f(ds, "%s page", __wt_page_type_string(dsk->type)));
+    WT_ERR(ds->f(ds, "- %s page\n\t", __wt_page_type_string(dsk->type)));
     switch (dsk->type) {
     case WT_PAGE_BLOCK_MANAGER:
         break;
     case WT_PAGE_COL_FIX:
     case WT_PAGE_COL_INT:
     case WT_PAGE_COL_VAR:
-        WT_ERR(ds->f(ds, ", recno %" PRIu64, dsk->recno));
+        WT_ERR(ds->f(ds, "recno: %" PRIu64 ", ", dsk->recno));
     /* FALLTHROUGH */
     case WT_PAGE_ROW_INT:
     case WT_PAGE_ROW_LEAF:
-        WT_ERR(ds->f(ds, ", entries %" PRIu32, dsk->u.entries));
+        WT_ERR(ds->f(ds, "entries: %" PRIu32 ", ", dsk->u.entries));
         break;
     case WT_PAGE_OVFL:
-        WT_ERR(ds->f(ds, ", datalen %" PRIu32, dsk->u.datalen));
+        WT_ERR(ds->f(ds, "datalen: %" PRIu32 ", ", dsk->u.datalen));
         break;
     default:
         WT_ERR(__wt_illegal_value(session, dsk->type));
     }
 
     if (F_ISSET(dsk, WT_PAGE_COMPRESSED))
-        WT_ERR(ds->f(ds, ", compressed"));
+        WT_ERR(ds->f(ds, "page_state: compressed, "));
     if (F_ISSET(dsk, WT_PAGE_ENCRYPTED))
-        WT_ERR(ds->f(ds, ", encrypted"));
+        WT_ERR(ds->f(ds, "page_state: encrypted, "));
     if (F_ISSET(dsk, WT_PAGE_EMPTY_V_ALL))
-        WT_ERR(ds->f(ds, ", empty-all"));
+        WT_ERR(ds->f(ds, "page_state: empty-all, "));
     if (F_ISSET(dsk, WT_PAGE_EMPTY_V_NONE))
-        WT_ERR(ds->f(ds, ", empty-none"));
+        WT_ERR(ds->f(ds, "page_state: empty-none, "));
 
-    WT_ERR(ds->f(ds, ", generation %" PRIu64 "\n", dsk->write_gen));
+    WT_ERR(ds->f(ds, "generation: %" PRIu64 "\n", dsk->write_gen));
```

Comment:
could we change the name from `generation` to `write_gen`?

Reply:
I was going to keep it as is to avoid merge conflicts since @adelinexchen is making that change